src/pages/home/home.scss

```diff
@@ -0,0 +1,74 @@
+@import "~theme/base/variables";
+
+.homeContainer {
+  height: 100%;
+}
+
+.navigationBar {
+  padding: 1.25em 0em;
+}
+
+.navigationBar .leftNavLinks {
+  margin-left: 4em;
+}
+
+.navigationBar .navLinks {
+  display: inline-block;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #179A6F;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 1rem;
+  margin-right: 2em;
+}
+
+.mainHeaderBackground {
+  background: url('../../assets/main-header-background.png') no-repeat;
+  height: 100%;
+  width: 100%;
+  background-size: cover;
+  position: absolute;
+  z-index: -1; 
+}
+
+.mainHeaderContent {
+  display: flex;
+  margin-left: 4em;
+}
+
+.mainHeaderContent > div {
+  flex: 1;
+}
+
+.mainHeaderContent > div .header {
+  font-family: $font-header;
+  font-size: 4.25rem;
+  color: #ffffff;
+}
+
+.mainHeaderContent > div .header:nth-child(1) {
+  margin-bottom: 0px;
+}
+
+.mainHeaderContent > div .header:nth-child(2) {
+  margin-top: 0px;
+}
+
+.mainHeaderContent > div h4 {
```

Comment:
When it comes to styling html elements, it's ok to nest them.

```
.mainHeaderContent {
  h4 { }
  p {
     span {}
  }
}
```

But nesting classnames is not necessary, let's avoid things like this: 

```
.someClassname {
   .somethignElse {

    }
}
```

Instead do this:
```
.someClassname {
  //...
}

.somethignElse {
  //...
}
```

Reply:
Sure, pushing it in a bit.